cfgov/v1/views/reports.py

```diff
@@ -151,3 +182,51 @@ def get_queryset(self):
             .order_by("-created_at")
             .prefetch_related("tags")
         )
+
+
+class EnforcementActionsReportView(ReportView):
+    header_icon = "doc-full"
+    title = "Enforcement actions report"
+
+    list_export = [
+        "title",
+        "content",
+        "categories.all",
+        "court",
+        "docket_numbers.all",
+        "initial_filing_date",
+        "statuses.all",
+        "products.all",
+        "url",
+    ]
+    export_headings = {
+        "title": "Title",
+        "content": "Content",
+        "categories.all": "Forum",
+        "court": "Court",
+        "docket_numbers.all": "Docket Numbers",
+        "initial_filing_date": "Initial Filling",
+        "statuses.all": "Statuses",
+        "products.all": "Products",
+        "url": "URL",
+    }
+
+    custom_field_preprocess = {
+        "content": {"csv": process_content},
+        "categories.all": {"csv": process_categories},
+        "docket_numbers.all": {"csv": process_docket_numbers},
+        "statuses.all": {"csv": process_statuses},
+        "products.all": {"csv": process_products},
```

Comment:
With my other suggestions above, I think this can become:

```suggestion
        "content": {"csv": process_enforcement_action_page_content},
        "categories.all": {"csv": process_categories},
```